File: Code/Components/DanpheEMR.ServerModel/EmergencyModels/EmergencyDischargeSummaryVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.EmergencyModels
{
    public class EmergencyDischargeSummaryVM
    {
        public EmergencyPatientModel EmergencyPatient { get; set; }
        public VitalsModel Vitals { get; set; }
        public EmergencyDischargeSummaryModel DischargeSummary { get; set; }

        public string VisitCode { get; set; }

        public List<string> LabOrders { get; set; }
        public List<string>  ImagingOrders { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/emergency/shared/emergency-discharge-summaryVM.ts
import {
    NgForm,
    FormGroup,
    FormControl,
    Validators,
    FormBuilder
} from '@angular/forms';
import { EmergencyPatientModel } from './emergency-patient.model';
import { Vitals } from '../../clinical/shared/vitals.model';
import { EmergencyDischargeSummary } from './emergency-discharge-summary.model';

export class EmergencyDischargeSummaryVM {
    public EmergencyPatient: EmergencyPatientModel = new EmergencyPatientModel();
    public Vitals: Vitals = new Vitals();
    public DischargeSummary: EmergencyDischargeSummary = new EmergencyDischargeSummary();

    public VisitCode: string = null;

    public LabOrders: Array<string> = [];
    public ImagingOrders: Array<string> = [];
}

